proto/interchain_security/ccv/consumer/v1/genesis.proto

```diff
@@ -0,0 +1,92 @@
+syntax = "proto3";
+
+package interchain_security.ccv.consumer.v1;
+
+option go_package = "github.com/cosmos/interchain-security/v3/x/ccv/consumer/types";
+
+import "interchain_security/ccv/v1/shared_consumer.proto";
+import "ibc/lightclients/tendermint/v1/tendermint.proto";
+
+import "gogoproto/gogo.proto";
+import "interchain_security/ccv/v1/wire.proto";
+import "google/protobuf/timestamp.proto";
+import "tendermint/abci/types.proto";
+
+
+
+
+// GenesisState defines the CCV consumer genesis state
+//
+// Note: this type is only used on consumer side and references shared types with
+// provider
+message GenesisState {
```

Comment:
In https://github.com/cosmos/interchain-security/pull/1288 we renamed the consumer's GenesisState struct to `ConsumerGenesisState`, is there a reason why we change it back in this PR?

Reply:
We changed it in the shared type namespace to make it clear that it's consumer related. This type is the result of the split and resides  in the consumer namespaces so no need to repeat that in the name. The shared one kept it's name